src/rand.rs

```diff
@@ -156,7 +156,7 @@ mod tests {
 
     #[test]
     fn test_rand_bytes() {
-        let s = rand_bytes(8);
-        assert_eq!(8, s.len());
+        let s = rand_bytes(7);
```

Comment:
Let's add multiple test scenarios here. Can you instead do a for from 0 to 8 and check that rand_bytes(i).len() is i?

Reply:
Absolutely!